src/internal/m365/collection/groups/backup.go

```diff
@@ -174,10 +173,9 @@ func populateCollections[C graph.GetIDer, I groupsItemer](
 			continue
 		}
 
-		added := str.SliceToMap(maps.Keys(addAndRem.Added))
 		removed := str.SliceToMap(addAndRem.Removed)
 
-		cl.Add(count.ItemsAdded, int64(len(added)))
+		cl.Add(count.ItemsAdded, int64(len(addAndRem.Added)))
 		cl.Add(count.ItemsRemoved, int64(len(removed)))
```

Comment:
should this be `addAndRem.Removed`?

Reply:
Should have the same desired effect. `removed` is a map , `addAndRem.Removed` is a string slice. Hypothetically, if there was duplication, `len(map)` i.e. `len(removed)` would still be a better choice.